activerecord/lib/active_record/enum.rb

```diff
@@ -121,6 +121,8 @@ def inherited(base) # :nodoc:
       super
     end
 
+    # NOTE: Is above ActiveRecord::ModelSchema in the inheritance hierarchy
+    # such that it is called by +super+, not by any callbacks
     def load_schema! # :nodoc:
```

Comment:
This was surprising. It actually above ActiveRecord::ModelSchema, which is why ActiveRecord::ModelSchema.load_schema! calls super and this doesn't